NodeJS/__integration__/AgentClientCrud.it.ts

```diff
@@ -31,9 +31,8 @@ describe("AgentClient", () => {
    }, testTimeOut)
 
    test("createAgent", async () => {
-      let skillPriority = { skillNumber: 100, skillPriority: 2 }
       let username = "ddksgy3dnr" // randomString(10)
-      let result = await agentClient.createAgentAndStation(username, "Passw0rd!", [skillPriority])
+      let result = await agentClient.createAgentAndStation(username, "Passw0rd!")
```

Comment:
nit: this should be `const` as it doesn't change

Reply:
There are a whole bunch of these, so I'll stop pinging on them. but if you want a super fast and easy way to fix them all at once, you can update your `.eslintrc` file with:

```
{
  "rules": {
    "prefer-const": "error"
  }
}
```

And then run `npm run lint --fix` and it'll auto fix all of those for you.

Reply:
I have tslint configured. Still trying to get used to all the rules there.  But I tried --fix and it should take care of converting let to const.